spec/classes/mod/jk_spec.rb

```diff
@@ -115,7 +115,7 @@
 
         it_behaves_like 'minimal resources'
         it do
-          is_expected.to contain_file("/etc/httpd/conf.d/jk.conf")
+          is_expected.to contain_file("#{:mod_dir}/jk.conf")
```

Comment:
Ah, that should be it! I was using the path, when I should be using the resource title. I'll start over pushing very little at a time, so we can get somewhere. Thanks for the hint!